zerver/tests/test_message_flags.py

```diff
@@ -23,14 +23,15 @@
 from zerver.lib.test_classes import ZulipTestCase
 from zerver.lib.test_helpers import get_subscription, timeout_mock
 from zerver.lib.timeout import TimeoutExpiredError
-from zerver.lib.user_topics import add_topic_mute
+from zerver.lib.user_topics import set_user_topic_visibility_policy
```

Comment:
I think it'd be better to have these code paths call `do_set_user_topic_visibility_policy`; the `lib` functions are meant to be internal interfaces for just changing the database (but not sending events), and shouldn't generally be called elsewhere.

I'll propose changing their name as well in another comment.

Reply:
The current state of codebase doesn't follow this pattern.

Added a prep commit (2nd commit) -- This commit makes sure, `add_topic_mute` and `remove_topic_mute` is not directly called, instead `do_mute_topic` and `do_unmute_topic` is used.

Later, commits replaces `do_mute_topic` and `do_unmute_topic` with `do_set_user_topic_visibility_policy`.
Hence,
> it'd be better to have these code paths call do_set_user_topic_visibility_policy

is implemented.